CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.0-rc.3 (2018-11-20)
+
+- Updated flyout component to not close when an overlay above it is clicked. [#3](https://github.com/blackbaud/skyux-flyout/pull/3)
```

Comment:
"Fixed flyout component to not close when..."

Reply:
done.